content/getting-started/quick-start-my-first-webhook/step-2.md

```diff
@@ -0,0 +1,139 @@
+# _Step 2_ | Create your own Symfony app to display event subscriptions data 
+
+[//]: <> (Hey Akeneo colleagues, here is a repo with the working code if you need to test https://github.com/akeneo/events-api-app.)
```

Comment:
It is a hack. It will not be displayed on the page either by the inspector.

Reply:
Since the source code of the doc is accessible to everyone it's not really hidden, but I don't know if it matters